conbench/tests/api/test_runs.py

```diff
@@ -231,6 +231,27 @@ def test_closest_commit_different_machines_should_not_prefer_test_runs_as_baseli
         response = client.get(f"/api/runs/{contender_run.id}/")
         self.assert_200_ok(response, _expected_entity(contender_run, baseline_run.id))
 
+    def test_baseline_run_always_on_default_branch(self, client):
+        self.authenticate(client)
+        _, benchmark_results = _fixtures.gen_fake_data()
+
+        successes = 0
+        for benchmark_result in benchmark_results:
+            response = client.get(f"/api/runs/{benchmark_result.run_id}/")
+            assert response.status_code == 200, response.status_code
+            baseline_run_link = response.json["links"].get("baseline")
+            if baseline_run_link:
+                baseline_run_response = client.get(
+                    baseline_run_link.replace("http://localhost", "")
+                )
```

Comment:
note to self: trying to be/do [HATEOAS](https://en.wikipedia.org/wiki/HATEOAS)  is always a little weird :D It never lives up to its promises.